### ISSUE TITLE
Commands reward uses the players permission.

### ISSUE THREAD

**Issue by MintyMint** [CitizensDev/Citizens#517]
This reward is pointless if it can only issue commands that the player themselves can use. This really needs to be fixed as he vastly expands the kinds of rewards quests can give...

**Comment by Jogeta** [comment]
also had this issue, would be great to be able to do /pex user player add permission

**Comment by l1amm** [comment]
I agree with this; It would give the commands rewards a million more possible uses.
